samples/nrf9160/modem_shell/README.rst

```diff
@@ -36,6 +39,11 @@ LTE link control changes and queries the state of the LTE connection.
 Many of the changes are applied when going to online mode for the next time.
 You can store some link subcommand parameters into settings, which are persistent between sessions.
 
+3GPP Release 14 features are enabled by default, i.e., they are set when going into normal mode and this applies to bootup too.
+You can see the list of supported features in `3GPP Release 14 features AT command`_.
+3GPP Release 14 features can be disabled when going into normal mode by using ``link funmode --normal_no_rel14`` command and
+they can be disabled during autoconnect in bootup by using ``link nmodeauto --enable_no_rel14`` command.
```

Comment:
Should these be on the same line, although it's quite long?

Reply:
Just following the instructions given by technical writers. 😄 Maybe the sentence is too long and should be split in two?

Reply:
If they want long sentences not to be split then I'll make it long but let's see what they say.